asv_bench/benchmarks/indexing.py

```diff
@@ -218,8 +226,14 @@ def time_multiindex_get_indexer(self):
                       (0, 16), (0, 17), (0, 18),
                       (0, 19)], dtype=object))
 
+    def time_multiindex_large_get_loc(self):
+        self.mi_large.get_loc((999, 19, 'Z'))
+
+    def time_multiindex_med_get_loc(self):
+        self.mi_med.get_loc((999, 9, 'A'))
+
     def time_multiindex_string_get_loc(self):
-        self.mistring.get_loc((999, 19, 'Z'))
+        self.mi_small.get_loc((99, 'A'))
```

Comment:
the index has actually 3 levels (did that change? in the beginning I think it had only 2 levels). 
But so with the result it is not a full key, so I suppose takes a different code path? (maybe also useful to benchmark, but not what we want in this PR?)